src/settings.py

```diff
@@ -93,6 +93,29 @@ def save_splanes_to_file(splanes, filename):
         for p in splanes:
             out.write(p.toFile() + '\n')
 
+def get_version(settings_filename):
+    with open(settings_filename, "r") as settings_file:
+        settings = yaml.safe_load(settings_file)
+
+    version = settings["common"]["version"]
```

Comment:
Better to use `get` for dictionaries, at least for `"version"` or wrap with try/except because it will be sad to crash here